includes/class-stats.php

```diff
@@ -17,6 +17,9 @@
 class Stats {
 	use Singleton;
 
+	const TABLE       = 'wpjm_stats';
```

Comment:
IMHO it is a good idea to have the `CACHE_GROUP` constant, but I'm a bit :thinking:  about having the `TABLE` constant too. It may lead to people trying to use it when the correct is to use the `$wpdb->wpjm_stats` alias.

WDYT about removing it?

Reply:
Yup I added it in the first place expecting to use it, good point :) I made the `TABLE` constant private now.